.github/workflows/cc-payload-after-push-s390x.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
           - cc-qemu
           - cc-rootfs-image
           - cc-shim-v2
+          - cc-virtiofsd
```

Comment:
What are you using for file sharing as part of s390x?  9p?
Is the default reflected as part of the shim-v2 Makefile?

Reply:
yep, we got back to the default thanks to https://github.com/kata-containers/kata-containers/pull/5523

Reply:
I've confirmed that the e2e [test](https://github.com/confidential-containers/operator/blob/main/tests/e2e/run-local.sh#L104) works on s390x with `cc-virtiofsd` via operator:

```
INFO: Running non-TEE tests for kata-qemu                                                                                                                                                                          
1..7                                                                                                                                                                                                               
ok 1 [cc][agent][kubernetes][containerd] Test can pull an unencrypted image inside the guest                                                                                                                       
ok 2 [cc][agent][kubernetes][containerd] Test can pull a unencrypted signed image from a protected registry                                                                                                        
ok 3 [cc][agent][kubernetes][containerd] Test can pull an unencrypted unsigned image from an unprotected registry                                                                                                  
ok 4 [cc][agent][kubernetes][containerd] Test can pull an encrypted image inside the guest with decryption key                                                                                                    
ok 5 [cc][agent][kubernetes][containerd] Test cannot pull an encrypted image inside the guest without decryption key 
```